packages/wasm-dpp/src/identity.rs

```diff
@@ -15,18 +15,14 @@ use crate::utils::to_vec_of_serde_values;
 use crate::IdentityPublicKeyWasm;
 use crate::MetadataWasm;
 
+use state_transition::*;
+
+mod state_transition;
```

Comment:
I think it would be nice to make it `pub mode`, so the import path would follow JS file path